src/main/java/frc/robot/Constants.java

```diff
@@ -596,4 +597,39 @@ public static final class CraneConstants {
     public static final Translation2d kPositionIntake =
       new Translation2d(Units.degreesToRadians(32.042), 0.6824);
   }
+
+  public static final class ClimberConstants {
+    public static final boolean kEnable = false;
+    public static final int kMotorID = 16;
+    public static final double kHardMin = 0.0; //TODO: Set.
+    public static final double kMaxSpeed = 0.5; //TODO: Set. This number is from ACDC.
+    public static final double kMaxExtendedAngle = Math.toRadians(90); //TODO: Set.
+    public static final double kMinRetractedAngle = Math.toRadians(0); //TODO: Set
+    public static final double kConstrainedRange = Math.toRadians(0.02); //TODO: Set.
+    public static final double kRecalibratingSpeed = 0.5; //TODO: Set.
+
+    //TODO: find gear ratio and meters per rotation
+    public static final double kGearRatio = 140.0;
+    // With no gear reduction each motor rotation moves the climber 0.012 meters
```

Comment:
Hmm, with no gear reduction, the climber would spin in a full 360 degree loop, were there no obstructions. We're dealing in angles here rather than distances. I think this should be `... kAngleConversionFactor = 2.0 * Math.PI / kGearRatio;`.